doc-tool/test/dotty/tools/dottydoc/DottyDocTest.scala

```diff
@@ -96,7 +98,8 @@ trait DottyDocTest extends MessageRendering {
   def checkFiles(sources: List[String])(assertion: (Context, Map[String, Package]) => Unit): Unit = {
     val c = compilerWithChecker(assertion)
     val run = c.newRun
-    run.compile(sources)
+    val files = sources.map(fileFromPath)
+    run.compile(files)
```

Comment:
Since this patterns occurs a few times, it seems worth having a method to do this in Run.scala (named `compileFromPaths` or something)

Reply:
Not needed anymore.
This pattern happens only in the tests and only 2 times.